lib/pages/AppConnectorConfig/AppConnectorConfig.tsx

```diff
@@ -0,0 +1,331 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory, useParams } from 'react-router'
+import {
+  Box,
+  Typography,
+  useTranslation,
+  TextField,
+  Grid,
+  Icon,
+  Switch,
+  FormControlLabel,
+  Button,
+  Divider,
+} from '@apisuite/fe-base'
+import {
+  getAppConnectorConfigAction,
+  getAppConnectorSubscriptionAction,
+  getAppDetailsAction,
+} from '../Marketplace/ducks'
+import useStyles from './styles'
+import appConnectorConfigSelector from './selector'
+import { Alert } from '@material-ui/lab'
+
+const AppConnectorConfig: React.FC = () => {
+  const classes = useStyles()
+
+  const {
+    selectedAppDetails,
+    appConnectorConfigDetails,
+    appConnectorSubscriptionDetails,
+    appConnectorSubscribed,
+  } = useSelector(appConnectorConfigSelector)
+
+  const dispatch = useDispatch()
+
+  const trans = useTranslation()
+
+  const t = (string: string, ...args) => {
+    return trans.t(`extensions.marketplace.${string}`, ...args)
+  }
+
+  const history = useHistory()
+  // 1. Subscription logic for the currently selected Marketplace app
+
+  // Retrieves the app's ID from the browser window's URL
+  const { appID } = useParams<any>()
+
+  const getSubscriptionStatus = () =>
+    appConnectorConfigDetails.data.workerStatus !== 'stopped' &&
+    appConnectorSubscribed
+
+  const backToApp = () => {
+    history.push(
+      `${encodeURI('/marketplace/app-details/' + selectedAppDetails.id)}`
+    )
+  }
+
+  const [fieldValues, setFieldValues] = useState({
+    apiUrl: '',
+    polling_toggle: getSubscriptionStatus(),
+  })
+  console.log('field values:', fieldValues)
+
+  const renderFieldsRaw = (entries) => {
+    return entries.map((entry, key) => (
+      <React.Fragment key={key}>
+        <Grid item xs={5}>
+          <TextField
+            label={t('appMarketplace.appConnectorConfig.appField')}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            disabled={true}
+            style={{ width: 100 + '%' }}
+            value={entry}
+            variant="outlined"
+          />
+        </Grid>
+        <Grid item xs={2}>
+          <Icon
+            style={{
+              width: 100 + '%',
+              height: 100 + '%',
+              textAlign: 'center',
+              lineHeight: 50 + 'px',
+              color: '#62acee',
+            }}
+          >
+            arrow_right_alt
+          </Icon>
+        </Grid>
+        <Grid item xs={5}>
+          <TextField
+            label={t('appMarketplace.appConnectorConfig.apiField')}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            style={{ width: 100 + '%' }}
+            name={entry}
+            value={fieldValues[entry]}
+            onChange={changeHandler}
+            variant="outlined"
+          />
+        </Grid>
+      </React.Fragment>
+    ))
+  }
+
+  const changeHandler = (changeEvent) => {
+    console.log(changeEvent)
```

Comment:
Just a reminder so you don't do as I do all the time ahahah - **don't forget to remove any trailing console.log() calls**!